Makefile

```diff
@@ -1,36 +1,37 @@
 SHELL := /bin/bash
-WHEELHOUSE := ./wheelhouse
 PYVER := 3.6
 VENV := ./env$(PYVER)
 
-version = $(shell "bin/get-version.sh")
+version := $(shell "bin/get-version.sh")
 
-dist/spacy-$(version).pex : wheelhouse/spacy-$(version)-*.whl
-	pex -f ./wheelhouse --no-index --disable-cache -m spacy -o dist/spacy-$(version).pex spacy==$(version) jsonschema
-	chmod a+rx dist/spacy-$(version).pex
+dist/spacy-$(version).pex : wheelhouse/spacy-$(version).stamp
+	$(VENV)/bin/pex -f ./wheelhouse --no-index --disable-cache -m spacy -o $@ spacy==$(version) jsonschema
+	chmod a+rx $@
 
 dist/pytest.pex : wheelhouse/pytest-*.whl
-	$(VENV)/bin/pex -f ./wheelhouse --no-index --disable-cache -m pytest -o dist/pytest.pex pytest pytest-timeout mock
-	chmod a+rx dist/pytest.pex
+	$(VENV)/bin/pex -f ./wheelhouse --no-index --disable-cache -m pytest -o $@ pytest pytest-timeout mock
+	chmod a+rx $@
 
-wheelhouse/spacy-$(version)-%.whl : $(VENV)/bin/pex setup.py spacy/*.py* spacy/*/*.py*
+wheelhouse/spacy-$(version).stamp : $(VENV)/bin/pex setup.py spacy/*.py* spacy/*/*.py*
 	$(VENV)/bin/pip wheel . -w ./wheelhouse
 	$(VENV)/bin/pip wheel jsonschema spacy_lookups_data -w ./wheelhouse
+	touch $@
 
 wheelhouse/pytest-%.whl : $(VENV)/bin/pex
 	$(VENV)/bin/pip wheel pytest pytest-timeout mock -w ./wheelhouse
 
-$(VENV) : 
+$(VENV)/bin/pex :
 	python$(PYVER) -m venv $(VENV)
 	$(VENV)/bin/python -m pip install pex wheel
 
 .PHONY : clean test
 
 test : dist/spacy-$(version).pex dist/pytest.pex
-	PEX_PATH=dist/spacy-$(version).pex ./dist/pytest.pex --pyargs spacy -x
+	( . $(VENV)/bin/activate ; \
+	PEX_PATH=dist/spacy-$(version).pex ./dist/pytest.pex --pyargs spacy -x ; )
```

Comment:
Why is this necessary? Pex shouldn't require the virtualenv to run I don't think? Is it just that it's picking the wrong Python for you?

Reply:
I think the problem is that `pex` isn't necessarily installed in the local venv? I got `ModuleNotFoundError: No module named 'pex'` otherwise.